### PR TITLE
[DX-533] Add Quick Start Guide For Tyk OSS Gateway

### PR DESCRIPTION
**Pull request by dcs3spp**
[DX-533](https://tyktech.atlassian.net/browse/DX-533)

Add quick start guide for Tyk OSS GW

[Tyk OSS GW Quick Start Guide Refactored](https://deploy-preview-2960--tyk-docs.netlify.app/docs/nightly/deployment-and-operations/tyk-open-source-api-gateway/quick-start/)
[Docker Install Refactored ](https://deploy-preview-2960--tyk-docs.netlify.app/docs/nightly/tyk-oss/ce-docker/)

[DX-533]: https://tyktech.atlassian.net/browse/DX-533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+**Step 3 - Deploy Tyk Gateway and Redis**
+```bash
+docker-compose up
+```
+
+<br>
+
+{{< note success >}}
+**note**
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md]: should it be capital?

File: tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Test Installation
+
+Your Tyk Gateway is now configured and ready to use. Confirm this by checking against the ‘hello’ endpoint:
+
+```bash
+curl localhost:8080/hello
+```
+
+Output should be similar to that shown below:
+`{"status":"pass", "version": "v3.2.1", "description": "Tyk GW"}`
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md]:
```suggestion
```json
{"status": "pass", "version": "v5.1", "description": "Tyk GW"}
```
```

File: tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## Next Steps
+
+Next, visit [adding]({{< ref "getting-started/create-api" >}}) your first API to Tyk and follow the Open Source instructions.
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md]:
```suggestion
Next, [add your first API to Tyk ]({{< ref "getting-started/create-api" >}}) and follow the *Open Source* instructions.
```

File: tyk-docs/content/tyk-oss/ce-docker.md [hunk 4]
```diff
@@ -20,26 +18,16 @@ The quickest way to get started is using docker-compose. Visit our [Dockerhub](h
...
  - Redis   - required for all Tyk installations.
              Simple redis installation instructions are included below.
- - MongoDB - Required only if you chose to use the MongoDB Tyk pump with your Tyk OSS installation. Same goes with any [other pump]({{< ref "/content/tyk-stack/tyk-pump/other-data-stores.md" >}}) you choose to use.
-             
+ - MongoDB - Required only if you chose to use the MongoDB Tyk pump with your Tyk OSS installation. Same goes with any [other pump]({{< ref "tyk-stack/tyk-pump/other-data-stores.md" >}}) you choose to use.
+
+## How To Install?
+
 {{< tabs_start >}}
 {{< tab_start "Docker Compose" >}}
```
Comment (letzya) [inline tyk-docs/content/tyk-oss/ce-docker.md]: Are we removing the docker compose tab? and adding a small message of killerkoda in the quick start, at the bottom?

File: tyk-docs/data/menu.yaml [hunk 5]
```diff
@@ -81,6 +73,15 @@ menu:
           path: /tyk-oss/ce-helm-chart-new
           category: Page
           show: True
+
+        - title: "Docker"
```
Comment (letzya) [inline tyk-docs/data/menu.yaml]: can we please put it first in the directory? then helm?

File: tyk-docs/content/tyk-oss/ce-docker.md (hunk 4 above)
Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md]: @letzya So suggesting remove the docker compose tab and keep the steps detailed in docker standalone tab with killerkoda at the bottom?

Currently PR has docker-compose and docker standalone tab. KillerKoda is at bottom of docker-compose tab currently in PR.

File: tyk-docs/data/menu.yaml (hunk 5 above)
Comment (dcs3spp) [reply tyk-docs/data/menu.yaml]: @letzya Done and pushed changes up to PR

File: tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/content/deployment-and-operations/tyk-open-source-api-gateway/quick-start.md]: @letzya Done and pushed changes to PR

File: tyk-docs/content/tyk-oss/ce-docker.md [hunk 6]
```diff
@@ -48,45 +36,57 @@ git clone https://github.com/TykTechnologies/tyk-gateway-docker
...
+
 {{< tab_end >}}
 {{< tab_start "Docker Standalone" >}}
 
 <br>
 
-### Step 1 - Let's create a network
+**Step 1 - Let's create a network**
 
 ```.bash
```
Comment (letzya) [inline tyk-docs/content/tyk-oss/ce-docker.md]:
```suggestion
```console
```

File: tyk-docs/content/tyk-oss/ce-docker.md [hunk 7]
```diff
@@ -48,45 +36,57 @@ git clone https://github.com/TykTechnologies/tyk-gateway-docker
...
 $ docker network create tyk
 ab1084d034c7e95735e10de804fc54aa940c031d2c4bb91d984675e5de2755e7
 ```
 
-### Step 2 - Deploy Redis into the network, with the `6379` port open
+**Step 2 - Deploy Redis into the network, with the `6379` port open**
 
 ```.bash
```
Comment (letzya) [inline tyk-docs/content/tyk-oss/ce-docker.md]:
```suggestion
```console
```

File: tyk-docs/content/tyk-oss/ce-docker.md [hunk 8]
```diff
@@ -100,14 +100,17 @@ $ docker run \
...
 
-Congratulations, you're done! Your Tyk Gateway is now configured and ready to use.
-Confirm this by checking against the 'hello' endpoint:
+Congratulations, you're done!
+
+## Test Installation
+
+Your Tyk Gateway is now configured and ready to use. Confirm this by making a network request to the 'hello' endpoint:
 
 ```bash
```
Comment (letzya) [inline tyk-docs/content/tyk-oss/ce-docker.md]:
```suggestion
```console
```

File: tyk-docs/content/tyk-oss/ce-docker.md (hunk 4 above)
Comment (letzya) [reply tyk-docs/content/tyk-oss/ce-docker.md]: agreed to remove the docker compose and move killerkoda to the quick start

Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md]: @letzya done and pushed the changes to the PR

File: tyk-docs/content/tyk-oss/ce-docker.md (hunk 6 above)
Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md]: @letzya Added and pushed change

File: tyk-docs/content/tyk-oss/ce-docker.md (hunk 7 above)
Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md]: @letzya Added and pushed change

File: tyk-docs/content/tyk-oss/ce-docker.md (hunk 8 above)
Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md]: @letzya Added and pushed change

File: tyk-docs/content/tyk-oss/ce-docker.md [hunk 9]
```diff
@@ -97,20 +55,35 @@ $ docker run \
   -v $(pwd)/apps:/opt/tyk-gateway/apps \
   docker.tyk.io/tyk-gateway/tyk-gateway:latest
 ```
-{{< tab_end >}}
-{{< tabs_end >}}
 
-Congratulations, you're done! Your Tyk Gateway is now configured and ready to use.
-Confirm this by checking against the 'hello' endpoint:
+Congratulations, you're done!
```
Comment (letzya) [inline tyk-docs/content/tyk-oss/ce-docker.md:59]: The killercoda should be at the bottom of the quick start, it's not related to the docker install

Comment (dcs3spp) [reply tyk-docs/content/tyk-oss/ce-docker.md:59]: Thanks @letzja. Have moved the killercoda section to the bottom of the quick start guide.

**Review by letzya — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**